shell/synchronize/etc/datf.cfg

```diff
@@ -1,6 +1,6 @@
 source_dir='/groups/solve-rd/prm10/sftp/datf'
-destination_dir='/groups/umcg-solve-rd/scr01/'
-destination_machine='umcg-${ATEAMBOTUSER}@cher-ami.hpc.rug.nl'
+destination_dir='groups/umcg-solve-rd/'
```

Comment:
Like Pieter mentioned on the other file. Is this line with missing starting '/' correct?

Reply:
Yes, is correct (rsync "module" syntax).